product_combination_exclude/security/product_attribute_exclude.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Graeme Gellatly
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+<data>
+
+    <record model="ir.model.access" id="product_attribute_exclude_access">
```

Comment:
because thats the way mrbob works and I prefer using tools that make things fast to do